examples/tasks/basic/example_task.py

```diff
@@ -148,9 +148,9 @@ def _prompt_for_save(self):
                     return self._prompt_for_save()
         return True
 
-    @on_trait_change("window:closing")
+    @observe("window:closing")
     def _prompt_on_close(self, event):
         """ Prompt the user to save when exiting.
         """
         if not self._prompt_for_save():
-            event.veto = True
+            event.new.veto = True
```

Comment:
same comment as earlier about replacing `event.new` with `window` to make the code more readable.